### PR TITLE
document how to submit yast in container

### PR DESCRIPTION
**Pull request by jreidinger**
## Problem

There is missing info about how to submit containerized yast.

## Solution

Write a document. Still not final as some OBS repos are not set up.

@jsrain please also review

### CODE REVIEW

**Comment by jsrain** [comment]
Looks good, I will get back to you when the OBS set-up is prepared (not yet)

File: SUBMITTING.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+- Container target project should be something under SUSE:ALP ( not yet created )
+- scripts devel project is common [YaST:Head](https://build.opensuse.org/project/show/YaST:Head)
+- scripts target project is common openSUSE:Factory from which it will be time to time synced to ALP.
+
+For scripts there are [ci in place](https://ci.opensuse.org/job/yast-yast-in-container-master/).
+
+### Notes
+
+Containers base image can be what fits us, as ALP should be able to run older and also newer code streams.
+So both SLE BCI or tumbleweed BCI can be used.
```
Comment (lslezak) [inline SUBMITTING.md:18]:
```suggestion
So both SLE BCI or Tumbleweed BCI can be used.
```

File: SUBMITTING.md [hunk 2]
```diff
@@ -0,0 +1,18 @@
+## Overview
+
+YaST in terminal is organized in this script to run it and containers
+that is used to run YaST.
+
+### OBS Repos
+
+- Container devel project is [YaST:Head:Containers](https://build.opensuse.org/project/show/YaST:Head:Containers)
+- Container target project should be something under SUSE:ALP ( not yet created )
+- scripts devel project is common [YaST:Head](https://build.opensuse.org/project/show/YaST:Head)
```
Comment (lslezak) [inline SUBMITTING.md:10]:
```suggestion
- Scripts devel project is common [YaST:Head](https://build.opensuse.org/project/show/YaST:Head)
```

File: SUBMITTING.md [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+- scripts target project is common openSUSE:Factory from which it will be time to time synced to ALP.
```
Comment (lslezak) [inline SUBMITTING.md:11]:
```suggestion
- Scripts target project is common openSUSE:Factory from which it will be time to time synced to ALP.
```

**Review by lslezak — APPROVED** [review]
LGTM, just small fixes

**Pull request merged**